src/widgets/header/ui/subaccount-selector.tsx

```diff
@@ -23,7 +24,11 @@ export const SubaccountSelector = observer(() => {
       <DropdownMenu>
         <DropdownMenu.Trigger>
           <Button>
-            <AddressViewComponent copyable={false} addressView={valueAddress} />
+            {isLoading ? (
+              <SpinnerIcon className='animate-spin' />
```

Comment:
Suggestion: When it loads, the `main acccount` state of the button is ~170px wide. Think we should make the button width in the loading state the same width so there is less screen jank as it transitions from loading to loaded.

Reply:
noting that a width of `~142px` fits perfectly on screen reloads, but I adjusted it to `170px` to match the button width.